### PR TITLE
Fix Icons Section (Home Page)

### PR DESCRIPTION
**Pull request by anshi05**
This PR addresses the issue  #20 where emoji icons were not rendering correctly on certain devices, displaying square boxes instead of the intended icons. To ensure consistent icon rendering across all platforms, the emoji icons have been replaced with the **react-icons** library, which provides reliable SVG icons.

### Changes:
- Replaced direct emoji icons (`🚀`, `🏠`, `🌞`, etc.) with SVG icons from the **react-icons** library.
- Updated the components to import relevant icons from **react-icons**.
- Adjusted the styles to ensure the new icons fit seamlessly within the UI.

### Testing:
- Verified the icons render correctly across different browsers and devices.
- Tested the visual layout to confirm it matches the original design.

### CODE REVIEW

**Review by bryanlundberg — CHANGES_REQUESTED** [review]
Hey @anshi05 Required changes should be on file `/pages/Home.tsx` line 101-123 👌

Actually, `AppLaunchComplete.tsx` can be deleted, it's not used on the website anymore.

**Comment by anshi05** [comment]
I'll do the required changes.

**Comment by anshi05** [comment]
I made the changes, you can check it now!

File: package.json [hunk 1]
```diff
@@ -23,7 +23,9 @@
     "lucide-react": "^0.435.0",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
+    "react-emoji-render": "^2.0.1",
     "react-hook-form": "^7.53.0",
+    "react-icons": "^5.3.0",
```
Comment (bryanlundberg) [inline package.json:28]: If these are not being used, please revert the changes and the `package-lock.json`

File: src/components/ui/AppLaunchComplete/AppLaunchComplete.tsx [hunk 2]
```diff
@@ -2,13 +2,14 @@ import { useState } from "react";
...
-  { id: 1, name: "Lanzamiento de aplicaciones", icon: "🚀" },
-  { id: 2, name: "Cocina Reno", icon: "🏠" },
-  { id: 3, name: "Hábitos diarios", icon: "🌞" },
-  { id: 4, name: "Recetas", icon: "🍔" },
-  { id: 5, name: "Trabajo de diseño", icon: "🎨" },
+  { id: 1, name: "Lanzamiento de aplicaciones", icon: <FaRocket /> },
+  { id: 2, name: "Cocina Reno", icon: <FaHome /> },
+  { id: 3, name: "Hábitos diarios", icon: <FaSun /> },
+  { id: 4, name: "Recetas", icon: <FaHamburger /> },
+  { id: 5, name: "Trabajo de diseño", icon: <FaPaintBrush /> },
```
Comment (bryanlundberg) [inline src/components/ui/AppLaunchComplete/AppLaunchComplete.tsx:12]: Could you please revert this change or remove the file and folder if not necessary? It seems this change might not be needed.

**Review by bryanlundberg — CHANGES_REQUESTED** [review]
Hey @anshi05 you did a great job.

Your implementation looks correct, and I have verified it on different machines, where the icons render as expected. I just have one request before we merge. Let's merge only the necessary changes.

File: package.json (hunk 1 above)
Comment (anshi05) [reply package.json:28]: Okk cool

File: src/components/ui/AppLaunchComplete/AppLaunchComplete.tsx (hunk 2 above)
Comment (bryanlundberg) [reply src/components/ui/AppLaunchComplete/AppLaunchComplete.tsx:12]: `/AppLaunchComplete/AppLaunchComplete.tsx` you can delete that or revert the change. Up to you.

File: package-lock.json
Comment (bryanlundberg) [inline package-lock.json:1]: Missing remove this changes, after this it's ready :)

Comment (anshi05) [reply package-lock.json:1]: how should I do that?

Comment (bryanlundberg) [reply package-lock.json:1]: The easiest way to avoid complicated commands should be creating a new PR with a clean branch and just commit again 

```
                  className="bg-[#F3F3F7] py-2 hover:bg-[#F3F3F7]"
                >
                  <p className="font-semibold text-gray-800">
                    <span className="me-3">&#128165;</span> App Launch
                  </p>
                </NavigationItem>
                <NavigationItem active={false}>
                  <p className="py-2 font-semibold text-gray-800 hover:bg-[#F3F3F7]">
                    <span className="me-3">&#127968;</span>Kitchen Reno
                  </p>       
                </NavigationItem>
                <NavigationItem active={false}>
                  <p className="py-2 font-semibold text-gray-800 hover:bg-[#F3F3F7]">
                    <span className="me-3">&#129496;</span>Daily Habits
                  </p>
                </NavigationItem>
                <NavigationItem active={false}>
                  <p className="py-2 font-semibold text-gray-800 hover:bg-[#F3F3F7]">
                    <span className="me-3">&#127828;</span>Recipes
                  </p>
                </NavigationItem>
                <NavigationItem active={false}>               
                  <p className="py-2 font-semibold text-gray-800 hover:bg-[#F3F3F7]">
                    <span className="me-3">&#9997;</span>Design Work
                  </p>                  
                </NavigationItem>                
              </div>

```

Comment (anshi05) [reply package-lock.json:1]: okk I'll create another PR
